### ISSUE TITLE
Assume https:// in search if no schema is provided

### ISSUE THREAD

**Issue by benoitjpnet** [pixelfed/pixelfed#1606]
I'm on cb62ab4eccb02ac268d361403a361b5f98ad3ef1.

Trying to get the last feature a.k.a remote follow.
Putting `mastodon.social/@pixelfed` or `@pixelfed@mastodon.social` in search bar return no results.

What I have in .env:

```
ACTIVITY_PUB=true
AP_OUTBOX=true
AP_INBOX=true
AP_SHAREDINBOX=true
AP_REMOTE_FOLLOW=true
REMOTE_FOLLOW=true
```
Based on https://github.com/pixelfed/pixelfed/blob/cc8d66c11b2fa12ceb768927ecc7c0a88e8838bf/config/federation.php#L20

How I can debug that?

**Comment by benoitjpnet** [comment]
:ah: We need to add the full URL, so it's https://mastodon.social/@pixelfed.
Support for @user@instance.tld would be nice.

**Comment by Deuchnord** [comment]
I'm also having issues to find an account hosted on another instance.

* Instance : pix.deuchnord.fr
* Version: 0.10.0

I'm trying to follow the following account: `benoit@pix.benoit.fr` from `deuchnord@pix.deuchnord.fr`.

Tried to search the following strings in the search box on top of the UI:

* `benoit@pix.benpro.fr`
* `@benoit@pix.benpro.fr`
* `pix.benpro.fr/benoit`
* `https://pix.benpro.fr/benoit`

All these search have given no results. Furthermore, it looks like @benpro, who owns the `@benoit@pix.benpro.fr`, could find my account, but not follow it.

**Comment by michel-slm** [comment]
On the main pixelfed.social instance, if I search a remote user by their full URL e.g. https://pixelfed.de/NicolasPetton it does show up, but clicking on 'Follow' takes me to the remote URL instead of actually following the user. Is this expected?

**Comment by benoitjpnet** [comment]
On 26c1f6362a6c88dde59ed6785613b88bf82ae37d,  searching for `https://mastodon.social/@pixelfed` return no results. Regression?

**Comment by dansup** [comment]
@benpro I am following you from pixelfed.social, are you still experiencing this issue?
<img width="1126" alt="Screen Shot 2019-08-20 at 2 37 46 PM" src="https://user-images.githubusercontent.com/877217/63382565-337ed800-c358-11e9-988c-b9d31736ad56.png">

**Comment by benoitjpnet** [comment]
@dansup I don't see your follow.
But now searching "https://pixelfed.social/dansup" return your profile. Still, I hit the bug #1598.

**Comment by benoitjpnet** [comment]
Sorry, I see it. Everything looks good then.

**Comment by dansup** [comment]
Fixed in #2083
